tutorials/pl/basic-snap-usage-pl/basic-snap-usage.md

```diff
@@ -0,0 +1,329 @@
+---
+id: basic-snap-usage-pl
+summary: W tym poradniku zostaną poruszone podstawowe informacje dotyczące korzystania z pakietów snap na Twojej dystrybucji oraz korzyści wynikające z używania ich.
+categories: packaging
+language: pl
+status: Published
+feedback link: https://github.com/ubuntu/codelabs/issues
+tags: snap, usage, beginner, idf-2016, polish
+difficulty: 1
+translator: Marcin Mikołajczak
+published: 2017-11-28
+
+---
+
+# Podstawy korzystania ze snapów
+
+## Wprowadzenie
+Duration: 1:00
+
+: Switch language: PL | [EN](tutorial/basic-snap-usage)
+
+Witamy w świecie snapów! W tym poradniku opisane zostaną podstawy korzystania z pakietów [snap] na Twojej dystrybucji Linuxa oraz poznasz korzyści wynikające z ich działania.
+
+
+![IMAGE](./snaps-hero.png)
+
+
+### Czego się nauczysz?
+
+  - Jak zainstalować usługę snapd na Twoim systemie.
+  - Jak szukać oprogramowania.
+  - Jak sprawdzić listę zainstalowanego oprogramowania.
+  - Jak aktualizowane są snapy.
+  - Czym są kanały snap i jak korzystać z nich.
+  - Jak wrócić do poprzedniej wersji snapa.
+
+Jeżeli wydaje Ci się to natłokiem informacji, przekonasz się, że jest inaczej. To wszystko jest bardzo proste dzięki prostocie interfejsu wiersza poleceń snapa.
+
+### Czego potrzebujesz?
+
+  - Dowolnej wspieranej dystrybucji systemu GNU/Linux (przeczytaj następny krok, aby poznać szczegóły).
+  - Podstawowej wiedzy dotyczącej korzystania z wiersza poleceń.
+
+Ten poradnik jest skupiony na poleceniu snap. Możesz po prostu skopiować i wkleić te polecenia.
+
+Survey
+: Jak wykorzystasz ten poradnik?
+ - Tylko przeczytam go
+ - Przeczytam i wykonam ćwiczenia
+: Jakie jest Twoje obecne doświadczenie?
+ - Nowicjusz
+ - Średniozaawansowany
+ - Doświadczony
+
+## Konfiguracja
+Duration: 2:00
+
+### Instalacja snapd
+`snapd` jest usługą, która działa na Twoim urządzeniu i monitoruje zainstalowane snapy, współdziała ze sklepem i dostarcza polecenie snap z którego będziesz korzystać. Instalacja na wymienionych poniżej dystrybucjach jest bezproblemowa.
+
+
+![IMAGE](./hero-artwork.png)
+
+
+
+### Różne dystrybucje:
+
+**Arch**
+
+```bash
+$ sudo pacman -S snapd
```

Comment:
This is actually confusing and wrong because the Arch maintainer has gone AWOL and we don't have a way to update the package. CC @bboozzoo who is both an Arch user, a snapd developer, a Canonical employee and a Polish speaker.

Reply:
Not sure how to work around this. Guess we could try reaching out to the maintainer once more. FWIW there's a `snapd-git` package in AUR. It's also mentioned in Archwiki Snapd page right here: https://wiki.archlinux.org/index.php/snapd

The user can install the package using one of the AUR helpers (`pacaur`, `yaourt` or other) or manually by downloading the package files.